catroid/src/main/java/org/catrobat/catroid/common/BroadcastMessageContainer.java

```diff
@@ -38,7 +42,8 @@ public BroadcastMessageContainer() {
 	}
 
 	public void update() {
-		Set<String> usedMessages = ProjectManager.getInstance().getCurrentlyEditedScene().getBroadcastMessagesInUse();
+		final Lazy<ProjectManager> projectManager = inject(ProjectManager.class);
```

Comment:
Please dont make variables lazy when they are accessed in the same function directly